service/controller/resource/monitoring/scrapeconfigs/resource.go

```diff
@@ -237,6 +237,12 @@ func hasPrometheusAgent(ctx context.Context, ctrlClient client.Client, cluster m
 				return false, microerror.Mask(err)
 			}
 			return version.GTE(gcpAgentVersion), nil
+		case "capa":
+			gcpAgentVersion, err := semver.Parse("0.11.0")
```

Comment:
Can you rename that to capaVersion?

Let's check with @erkanerol if we have a working release for capv/vcd so we Can support all providers starting now

Reply:
I updated the variable's name to capaAgentVersion. Is that ok for you ?

Reply:
That's good ;)